docs/changelog.rst

```diff
@@ -37,6 +37,21 @@ See :ref:`upgrade-guide-2.6` for help upgrading to the new version.
 * ``py::memoryview``  update and documentation.
   `#2223 <https://github.com/pybind/pybind11/pull/2223>`_
 
+* The Python package was reworked to be more powerful and useful.
+  `#2433 <https://github.com/pybind/pybind11/pull/2433>`_
+
+  * :ref:`build-setuptools` is easier thanks to a new
+    ``pybind11.setup_helpers`` module, which provides utilities to use
+    setuptools with pybind11, and can be used via PEP 518, ``setup_requires``,
+    or by directly copying ``setup_helpers.py`` into your project.
+
+  * CMake configuration files are now included in the Python package. Use
+    ``pybind11.get_cmake_dir()`` or ``python -m pybind11 --cmakedir`` to get
```

Comment:
This is true for `python -m pip`, as well. I will sometimes use `pip` out of laziness, but in CI, etc, it's better to write it out. Especially in the pip 9 -> 10 transition, where they broke the distribution `pip` script and didn't have the permissions to overwrite it.

Reply:
It is rather tempting to put this into the docs, though, since `pybind3 -m pybind11` is both more specific (3) and longer for an already long line...

Reply:
Would it make sense to add a "--cflags" option? And a `--extension-suffix` option possibly, to abstract away `python3-config --extension-suffix` into a form that doesn't require the note:

> The flags given here assume that you're using Python 3. For Python 2, just change the executable appropriately (to ``python`` or ``python2``).

Or the note:

> Note that Python 2.7 modules don't use a special suffix, so you should simply use ``example.so`` instead of ``example`python3-config --extension-suffix```. Besides, the ``--extension-suffix`` option may or may not be available, depending on the distribution; in the latter case, the module extension can be manually set to ``.so``.

Let's not do that in this PR, though.